modules/power-monitoring-hardware-virtualization-support.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * power_monitoring/power-monitoring-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="power-monitoring-hardware-virtualization-support_{context}"]
+= {PM-kepler} hardware and virtualization support
+
+{PM-kepler} is the key component of {PM-shortname} that collects real-time power consumption data from a node through one of the following power estimation methods:
+
+Kernel Power Management Subsystem (preferred)::
```

Comment:
```suggestion
Kernel Power Management Subsystem (supported)::
```

Reply:
Just noting that one was saying "preferred" and the other nothing. My suggestion would be to align them to either say nothing in parenthesis, or state that one is supported and the other unsupported (not preferred!)